src/components/datepicker/datePicker.js

```diff
@@ -385,7 +420,7 @@
         var maxDate = this.dateUtil.createDateAtMidnight(this.maxDate);
         this.ngModelCtrl.$setValidity('maxdate', date <= maxDate);
       }
-      
```

Comment:
i'd revert those empty lines and lines you didn't changed so when looking at things that change you won't show up